README.md

```diff
@@ -1,16 +1,54 @@
-# [Project Name]
+# Python Adventures
 
-[project description]
+![Alt title](bot/assets/title-art.png)
 
+Python Adventures is a discord game bot where you answer pythonic questions and gain rewards.
 
-## How to prepare the project?
+# Installation
+## Discord API Token
+What is a Discord API Token?
+A Discord API Token is a unique identifier used to authenticate requests to the Discord API. It acts as a password for your bot, allowing it to interact with Discord's servers, join channels, send messages, and perform other actions as defined by the Discord API.
 
-(optional) Create a virtual environment for this project.
+## How to Obtain a Discord API Token
+1. Create a New Application:
+  - Go to the [***Discord Developer Portal***](https://discord.com/developers/applications)
+  -  Click on ***New Application***
+  -   Give your application a name and click **Create**.
 
-1. Install the necessary packages: `pip install -r requirements.txt`
+2. Create a Bot:
+  - Navigate to the ***Bot*** section in the sidebar.
+  -  Click on "Add Bot" and confirm by clicking ***Yes, do it!***
 
-2. Create a `.env` at the root of the project, if not available.
+3. Copy the Token:
+   - Under the "TOKEN" section, click "Copy" to copy your bot's token.
+   - Keep this token secure and never share it publicly. If your token is exposed, you should regenerate it immediately.
 
-3. add or set `DISCORD_BOT_KEY=[your_bot_key]` with bot's key.
+For more information, Consult The [Discord Developer Portal](https://discord.com/developers/docs/intro) Documentation.
 
-4. You are ready to go!
+## Installation and Setup
+To get your Discord bot up and running, follow these steps:
+1. **Clone the repository**: `git clone github.com/gustavwilliam/cj11-mesmerizing-meteors && cd cj11-mesmerizing-meteors`
+
+2. **Install Requirements**:
+   > - You can create a virtual environment by `python -m venv .venv`
+   > - Activate it by `.venv/bin/activate` then pursue the setup
+
+Ensure you have `python` installed then install dependencies by `pip install -r requirements.txt`
+
+3. **Set Up Your Environment Variables**:
+   - Create a .env file in the root of your project and add your Discord API Token: `DISCORD_BOT_KEY="your-discord-token-here"`
+
```

Comment:
We should have a step for updating the emojis used in `config.py`. They should change the IDs to their own emojis' IDs, after they add the emojis under bot/assets/icons to a server or their bot application. Otherwise the bot won't work (unless its invited into our server where the emojis already exist)